spec/api/views/players/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+RSpec.describe Api::Views::Players::Index, type: :view do
+  let(:players) { build_players }
+  let(:exposures) { Hash[players: [players]] }
```

Comment:
I would rather see the explicit response hard coded here. This test would fail if `players` isn't what we expect it to be, which can happen more often than you might think.

Reply:
@jgayfer fixed.